packages/middleware-sdk-s3-control/src/process-arnables-plugin/getProcessArnablesPlugin.ts

```diff
@@ -6,7 +6,7 @@ import { updateArnablesRequestMiddleware, updateArnablesRequestMiddlewareOptions
 
 export const getProcessArnablesPlugin = (options: S3ControlResolvedConfig): Pluggable<any, any> => ({
   applyToStack: (clientStack) => {
-    clientStack.add(parseOutpostArnablesMiddleaware(options), parseOutpostArnablesMiddleawareOptions);
-    clientStack.add(updateArnablesRequestMiddleware(options), updateArnablesRequestMiddlewareOptions);
+    clientStack.addRelativeTo(parseOutpostArnablesMiddleaware(options), parseOutpostArnablesMiddleawareOptions);
+    clientStack.addRelativeTo(updateArnablesRequestMiddleware(options), updateArnablesRequestMiddlewareOptions);
```

Comment:
`parseOutpostArnablesMiddleaware` (I didn't want to fix the typo for backwards compatibility reasons) is now before serializer and `updateArnablesRequestMiddleware` is after serializer.

This prevents them from interfering with endpoint resolution by rewriting the inputs too early, but still lets them do their jobs.